quickstarts/rhdh-installation-via-helm.yaml

```
@@ -229,9 +229,9 @@ spec:
           2. Click on the Application launcher icon and check if your application menu item is shown.
           3. Verify that it opens your RHDH instance.
         failedTaskHelp: |-
-          Doublecheck your `ConsoleLink` resource or try to reload the OpenShift Console.
+          Double-check your `ConsoleLink` resource or try to reload the OpenShift Console.
       summary:
-        success: You have upgraded your RHDH instance!
+        success: You have added your Red Hat Developer Hub instance to the OpenShift Console menu!
```

```suggestion
        success: You have added your instance of Red Hat Developer Hub to the OpenShift Console menu!
```
Just a small reword suggestion